crates/aptos-rosetta/src/client.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::{AccountBalanceRequest, AccountBalanceResponse};
+use aptos_rest_client::aptos_api_types::mime_types::JSON;
+use aptos_types::{account_address::AccountAddress, chain_id::ChainId};
+use reqwest::{header::CONTENT_TYPE, Client as ReqwestClient};
+use url::Url;
+
+pub struct RosettaClient {
+    address: Url,
+    inner: ReqwestClient,
+}
+
+impl RosettaClient {
+    pub fn new(address: Url) -> RosettaClient {
+        RosettaClient {
+            address,
+            inner: ReqwestClient::new(),
+        }
+    }
+
+    pub async fn account_balance_simple(
```

Comment:
might it be more reasonable to mark this as accessible via test only?

Reply:
Yeah, we could do that, or I could just get rid of it, since it's causing confusion.